### DISCUSSION TITLE
Parallel map not work with Laravel

### DISCUSSION THREAD

**Discussion by scortes1989** [spatie/laravel-collection-macros#205] [Q&A]
`collect($documents)->parallelMap(function($file) use ($companyPackDocumentConfig) {
   Document::query()->create([
      'name'          => $file,
      'template_id'   => $companyPackDocumentConfig,
   ]);
});`

this does not work , error: Call to a member function connection() on null

**Comment by kmuenkel** [comment]
This may help: https://github.com/spatie/laravel-collection-macros/issues/108#issuecomment-1061329711

**Reply by Jampire** [reply]
How to see the content of that link?
